hack/verify-deps.sh

```diff
@@ -2,11 +2,16 @@
 set -euo pipefail
 
 function print_failure {
-  echo "There appear to be uncommitted changes to go.mod. Please run 'go tidy' and 'go vendor'."
+  echo "There are unexpected changes to the vendor tree following `go mod vendor` and `go mod tidy`. Please"
```

Comment:
Use single-quotes to prevent command expansion of ``` `go mod vendor` ``` and ``` `go mod tidy` ```.

Reply:
Fixed